conf/rest/9.12.0/certificate.yaml

```diff
@@ -0,0 +1,18 @@
+
+name:             Certificate
+query:            api/security/certificates
+object:           certificate
+
+counters:
+  - ^^uuid
+  - ^expiry_time             => expiry_time
+  - ^name
+  - ^svm.name                => svm
+
+export_options:
+  instance_keys:
+    - name
+    - svm
+    - uuid
+  instance_labels:
```

Comment:
As this is label, it's quite difficult to write alert, Let me explore to handle this.

Reply:
I need to use metric instead of label for this: 
This is the alert query for certificates expiring within 1 month:
`0 < (certificate_expiry_time{} - time()) < (30*24*3600)`

I will add sample warning alert as well for reference.